5.1.I_don't_have_vinaigrette.py

```diff
@@ -0,0 +1,27 @@
+import datetime
+import random
+
+
+
+date1 = input('Enter a date YYYY-MM-DD:')
+date2 = input('Enter another date YYYY-MM-DD:')
+date1=date1.split('-')
```

Comment:
Rename 'date' to start_date or end_date
 Variable names - https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0 - variable names, abbreviation reference is missing